README.md

```diff
@@ -269,7 +269,7 @@ $ kubectl create rolebinding kube-httpcache --clusterrole=kube-httpcache --servi
     apiVersion: v1
     kind: Service
     metadata:
-      name: cache-service
+      name: frontend-service
```

Comment:
Good catch! Although in this case, we'd need to also adjust the StatefulSet's `.spec.serviceName` (which currently also uses `cache-service`, IIRC).